pkg/kapp/cmd/appchange/sort_flag.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package appchange
+
+import "github.com/spf13/cobra"
+
+type SortFlag struct {
+	Sort string
+}
+
+func (s *SortFlag) Set(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&s.Sort, "sort", "s", "newest-first", "Sort app-changes list (allowed values : newest-first, oldest-first)")
+}
+
+func (s *SortFlag) IsSortByNewestFirst() bool {
```

Comment:
If we do go ahead with the string flag. We should definitely validate that the values are one of the allowed values, and error out otherwise while prompting the user with the acceptable values